### PR TITLE
Added the billing error codes

### PR DESCRIPTION
**Pull request by ahmed-abdulmoniem**
Added the billing error codes

### CODE REVIEW

File: codes.json [hunk 1]
```diff
@@ -364,6 +364,102 @@
       "en": "No groups names were provided."
     }
   },
+  "IncorrectNumber": {
+    "status": 402,
```
Comment (XinV) [inline codes.json]: Any reason we use 402 which is not used very ofter, although it says "Payment Required: This code is reserved for future use." ? I suggest we follow return 400 as other invalid request cases

Comment (ahmed-abdulmoniem) [inline codes.json]: Stripe is using 402 to indicate errors

From documentation:

> When a request is valid but does not complete successfully (e.g. a card is declined), we return a 402 error code.

Comment (XinV) [inline codes.json]: ok then

File: codes.json [hunk 2]
```diff
@@ -364,6 +364,102 @@
...
+  },
+  "CardDeclined": {
+    "status": 402,
+    "code": 4020050,
+    "error": "card_declined",
+    "description": {
+      "en": "The card was declined."
+    }
+  },
+  "Missing": {
```
Comment (cleondz) [inline codes.json]: could be MissingCard instead of just Missing???

File: codes.json [hunk 3]
```diff
@@ -364,6 +364,102 @@
...
+    "status": 402,
+    "code": 4020051,
+    "error": "missing",
+    "description": {
+      "en": "There is no card on a customer that is being charged."
+    }
+  },
+  "ProcessingError": {
```
Comment (cleondz) [inline codes.json]: Also here could be ProcessingCardError instead of just ProcessingError??

File: codes.json (hunk 2 above)
Comment (ahmed-abdulmoniem) [inline codes.json]: This is the same codes of Stripe with the same names. Just I am using the same notation of them.

File: codes.json (hunk 3 above)
Comment (ahmed-abdulmoniem) [inline codes.json]: This is the same codes of Stripe with the same names. Just I am using the same notation of them.

**Pull request merged**